src/utils.js

```diff
@@ -46,6 +46,17 @@ export const formatUTCDate = (participantDate, formatToYearMonthDay) => {
   return formatToYearMonthDay ? `${year}-${month}-${day}` : `${month}/${day}/${year}`;
 };
 
+/**
+ * Convert  human readable date to ISO8601 date
+ * @param {String} participantDate - human readable date
+ * @returns {String} - ISO8601 date
+ * 
+*/
+export const convertToISO8601 = (dateString) => {
```

Comment:
Not sure whether this one-liner should be a function or not: `(dateStr) => new Date(dateStr).toISOString()`